### PR TITLE
Refactor Exception class

### PR DESCRIPTION
**Pull request by youtr-s**

### CODE REVIEW

File: src/base/exception/KintoneAPIException.js [hunk 1]
```diff
@@ -1,79 +1,74 @@
...
+   * @return {ErrorResponse}
+   */
+  getErrorResponse() {
+    return this.errorResponse;
   }
-  /**
-     * Show Error
-     * @return {Error}
-     */
-  get() {
```
[the remark this answers is not in the document]
Comment (youtr-s) [reply src/base/exception/KintoneAPIException.js]: @trung-doan 
>*Note: we can remove it from the user guide and update sample code later.

Does this mean we should update function name and sample codes in the future but now keep them old style temporarily?

[the remark this answers is not in the document]
Comment (trung-doan) [reply src/base/exception/KintoneAPIException.js]: We should keep this function "get()" and add new function "getErrorResponse()".
So in source code, we will have 2 functions ( get() and getErrorResponse() ).
If developer have been using "get()" functions, their code can run normally.

In user guide, "get()" function will be removed.
In sample code of user guide, there are many functions using "get()", so we will replace them by "getErrorResponse()" later.

[the remark this answers is not in the document]
Comment (youtr-s) [reply src/base/exception/KintoneAPIException.js]: @trung-doan 
>We should keep this function "get()" and add new function "getErrorResponse()".

I see. I will re-implement new function "getErrorResponse()".

File: src/browser/module/connection/Connection.js [hunk 2]
```diff
@@ -35,12 +35,7 @@ export class Connection extends kintoneBaseJSSDK.Connection {
       return kintone.api(super.getUri(restAPIName), String(methodName).toUpperCase(), body).then((response) => {
         return response;
       }).catch(err => {
-        const error = {
-          response: {
-            data: err
-          }
-        };
-        throw error;
+        throw new kintoneBaseJSSDK.KintoneAPIException(err);
```
Comment (phongnhat19) [inline src/browser/module/connection/Connection.js]: The constructor of `KintoneAPIException` is implemented like this:
```javascript
constructor(message = '', errors = {}, ...args) {
    ///...
}
```
So I think the first argument should be a string. I'm not sure what is the expected result here but if you want to pass the `err` object, I think this line should be:
```javascript
throw new kintoneBaseJSSDK.KintoneAPIException('', err);
```

**Pull request merged**